Makefile

```diff
@@ -0,0 +1,11 @@
+NAME ?= inbd
+BUILD_DIR ?= output
+INBM_VERSION ?= 0.1.0
```

Comment:
how about `5.0.0-dev` for the version

Reply:
that sounds good, 5.0.0 matches the inbm v5